src/ProductConstructionService/ProductConstructionService.BarViz/Code/Helpers/JSRuntimeExtensions.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.JSInterop;
+
+namespace ProductConstructionService.BarViz.Code.Helpers;
+
+internal static class JSRuntimeExtensions
+{
+    public static async Task OpenNewWindow(this IJSRuntime jsRuntime, string uri)
```

Comment:
Blazor's navigation does not provide a call to open in new window?

Reply:
I was amazed but it does not! AI could not help and all StackOverflow threads suggested to write own function.